docs/source/geopmsession.1.rst

```diff
@@ -80,10 +82,30 @@ Options
 
     Stop the session when the given process PID ends
 
---print-header  .. _header option:
+--omit-header  .. _header option:
 
-    Print a CSV header before printing any sampled values
+    Do not print the CSV header before printing any sampled values
 
+--delimiter .. _delimiter DELIMITER .. option:
+
+    Delimiter used to separate values in CSV output
+
+--report-out .. _reportout REPORT_OUT option:
+
+    Output summary statistics into a yaml file. Note if ``--report-out=-``
+    is specified, the report will output to stdout.
+
+--trace-out .. _traceout TRACE_OUT option:
+
+    Output trace data into a CSV file. Note if ``--trace_out=-`` is specified,
+    the trace will output to stdout. To avoid gathering trace data,
+    set this parameter to ``/dev/null``
+
+--enable-mpi .. _enablempi option:
+
+    Gather reports over MPI and write to a single file. Append MPI rank to trace
+    output file if specified (trace output to stdout not permitted). Requires
```

Comment:
Append hostname instead of mpi rank to trace files when mpi enabled to be compatible with existing convention.